app/routes/user/endorsements.tsx

```diff
@@ -236,6 +238,11 @@ export function EndorsementRequestCard({
               name="requestorSub"
               value={endorsementRequest.requestorSub}
             />
+            <input
+              type="hidden"
+              name="requestorEmail"
+              value={endorsementRequest.requestorEmail}
+            />
```

Comment:
Can we get the user's email address from the session rather than injecting it into the form?

Reply:
The requestorEmail is not the same as the current user's in this case. When this form is submitted, the current user is the 'Endorser'

Reply:
I see. But can't we get this from DynamoDB?

Reply:
Hm I think we could, but we already do it in the loader method, with the getEndorsements('endorser') returning the email for display. I could update the updateEndorsementRequestStatus to just do a get on the dynamo table though, if that's better for this system

Reply:
Let's do it. We need to validate _anything_ that we get from a form submission, even if it's in a hidden field that we populated. In this case validation would be checking that the email matches what's in our database. So we should just get it from our database in the first place.